### PR TITLE
jira PDC-960, Order resources in client

### PR DESCRIPTION
**Pull request by bliuredhat**
Print resources in order when running  'pdc_client -s qa ' shell command.

jira: PDC_960

### CODE REVIEW

File: requirements/devel.txt [hunk 1]
```diff
@@ -10,6 +10,6 @@ setuptools
 lxml
 # model graph
 # requires: graphviz-devel
-pygraphviz
+pygraphviz<1.3
```
Comment (erichuanggit) [inline requirements/devel.txt]: I would suggest having a single patch to fix it, but not include this change in every patch.

**Comment by erichuanggit** [comment]
Generally looks good to me.

**Comment by simozhan** [comment]
Good to me

File: requirements/devel.txt (hunk 1 above)
Comment (bliuredhat) [inline requirements/devel.txt]: erichuanggit, yes， I will do；Thank you！

**Comment by lubomir** [comment]
Looks good to me. I think the commit message would deserve a bit more love.

**Pull request merged**